src/components/SkipNavigation/SkipNavigation.stories.mdx

```diff
@@ -80,7 +80,7 @@ The reason this is a navigation list of links is because the first main link
 points to the anchor of the main content, in this case `"#mainContent"`. The
 second link points to accessibility information on [NYPL.org/accessibility](https://nypl.org/accessibility).
 
-See the [Skip Navigation accessibility guide](https://nypl.github.io/nypl-design-system/storybook-static/?path=/story/accessibility-guide-skip-navigation--page)
+See the [Skip Navigation accessibility guide](https://nypl.github.io/nypl-design-system/reservoir/v1/?path=/story/accessibility-guide-skip-navigation--page)
```

Comment:
I thought we could use local paths to link to other Storybook pages.  In the Vercel build, this opens a new browser tab and I am wondering if that is because the link is using a fully qualified URL rater than a local path.

Reply:
Yeah, let's try relative.